### PR TITLE
Create aungkohtat.json

### PR DESCRIPTION
**Pull request by aungkohtat**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by sital002 — CHANGES_REQUESTED** [review]
Please put the file inside data folder instead of ```aungkohtat.json``` it should be ```data/aungkohtat.json```.
Please make the necessary changes to get your profile merged.

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
You now have two files.  Please check the "Files Changed" tab - the file called `aungkohtat.json` needs to be deleted.

File: data/aungkohtat.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Aung Ko Htet",
+  "bio": "I am passionate about all things cloud ☁️!",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/aungkohtet_",
+      "icon": "FaTwitter"
+    }
+    {
```
Comment (SaraJaoude) [inline data/aungkohtat.json:10]: add comma:
```suggestion
    },
    {
```

**Review by sital002 — COMMENTED** [review]
Since there has been no activity for 2 weeks I am closing this PR.
You can always open new PR with the lastest changes.

**Pull request closed without merging**